src/plugins/graphql.ts

```
@@ -0,0 +1,57 @@
+    path: '/graphql',
+    schema,
+    subscription: {
+      context: async (socket, request) =>
+        await createContext({
+          fastify,
+          isSubscription: true,
+          request,
+          socket,
+        }),
+      keepAlive: 1000 * 30,
+      onConnect: (data: { type: "connection_init"; payload: any; }) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)